crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -58,7 +58,7 @@ type ActiveJwkEdge {
 	cursor: String!
 }
 
-type Address implements ObjectOwner {
+type Address implements Iowner {
```

Comment:
interesting, wonder if this is due to async-graphql automagically converting IOwner to Iowner...

Reply:
Yes, `async-graphql` is converting it to `Iowner`.